srcpkgs/vscode/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'vscode'
 pkgname=vscode
-version=1.63.2
+version=1.65.2
 revision=1
-_electronver=13.6.1
+_electronver=13.6.7
 hostmakedepends="pkg-config python nodejs yarn tar git ripgrep"
```

Comment:
Can we use python3 here?

Reply:
I'll try it.

Reply:
works for me, so I'll include the update.